library/src/skins/default/src/sass/base/_defaults.scss

```diff
@@ -10,6 +10,10 @@ body{
   scroll-behavior: $scroll-behavior;
 }
 
+.progress {
+  background-color: var(--sakai-background-color-4) !important; // Override default progress bar background color
```

Comment:
is !important needed?

Reply:
since we need to override the default value of the progress bootstrap bar, this will sure apply on 100% of the cases

Reply:
but i don't think .progress is a bootstrap class? are you sure you need !important? did you test without it? please provide a screenshot

Reply:
@jumarub using !important is a last resort 
always prefer using css specificity first, see https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#tips_for_handling_specificity_headaches

Reply:
progress and progress-bar are bootstrap classes for the progress bar component, the sakai class and bootstrap class seem familiar
https://getbootstrap.com/docs/4.0/components/progress/

On the other hand I test it without the !important and it seems to work, so removing the !important is correct.